starter-kit: update to new onboarding steps (#208)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/structure.md

|   |   |   |__ webhook
|   |   |__ webhook
|   |   |   |__ <action>
|   |__ scripts
|   |   |__ commerce-event-subscribe

> |   |   |__ onboarding

Reviewer: If you've changed the folder structure, there are likely several references to files in the `/onboarding/` directory that may need to be changed to the `/scripts/onboarding` directory.

Author: Ouch! I missed some of those references. I've updated them in 0bd9f2d

Reviewer: @rojoangel was `/test/onboarding` also changed to `/test/data/onboarding`?

Author: Good catch. We moved the tests from `/test/onboarding` to `test/scripts`. I've updated the docs in this commit: e6e4232

`transformer.js` | Implements the logic to transform the incoming event payload to make it suitable for the target API being called to propagate the changes.
`validator.js` | Implements the logic to validate the incoming event payload.

### `scripts/onboarding` directory

> The `onboarding` directory contains the scripting needed to install and set up the starter kit. The `./onboarding/custom/starter-kit-registrations.json` configuration file creates all the registrations for all entities that are present in the repo's `app.config.yaml` file. You can edit this file to remove any unnecessary Commerce or back office registrations. [Onboarding](./create-integration.md#onboarding) describes the installation process.

Reviewer: ```suggestion
The `onboarding` directory contains the scripting needed to install and set up the starter kit. The `scripts/onboarding/custom/starter-kit-registrations.json` configuration file creates all the registrations for all entities that are present in the repo's `app.config.yaml` file. You can edit this file to remove any unnecessary Commerce or back office registrations. [Onboarding](./create-integration.md#onboarding) describes the installation process.
```
For example, this reference would need to be updated.

### `scripts/commerce-event-subscribe` directory

> The `commerce-event-subscribe` directory contains the scripting needed to subscribe the Commerce event provider created during onboarding to the Commerce events. The `./onboarding/custom/starter-kit-registrations.json` configuration file contains all the Commerce events to subscribe to. You can edit this file to add, remove or adapt the events needed by your project. [Onboarding](./create-integration.md#onboarding) describes the installation process.

Reviewer: ```suggestion
The `commerce-event-subscribe` directory contains the script to subscribe the Commerce event provider, which was created during onboarding, to the Commerce events. The `scripts/onboarding/custom/starter-kit-registrations.json` configuration file contains all the Commerce events you want subscribe to. Edit this file to add, remove, or modify the events necessary for your project. [Onboarding](./create-integration.md#onboarding) describes the installation process.
```
